dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java

```diff
@@ -0,0 +1,969 @@
+                    shortFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "H:mm:ss");
+                }
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ShortFormat=\"")
+                    .append(shortFormat)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("= $date.toDate(\"H:mm:ss\", \"")
+                    .append(shortFormat)
+                    .append("\"))");
+            } else if (field instanceof DateTimeField) {
+                String shortFormat = "";
+                String longFormat = "";
+                String dbFormat = "";
+                if (contFieldValueObject != null && contFieldValueObject instanceof Date) {
+                    shortFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "MM/dd/yyyy");
+                    longFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "MM/dd/yyyy H:mm:ss");
+                    dbFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "yyyy-MM-dd H:mm:ss");
+                }
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("= $date.toDate(\"yyyy-MM-dd H:mm:ss\", \"")
+                    .append(dbFormat)
+                    .append("\"))");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ShortFormat=\"")
+                    .append(shortFormat)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("LongFormat=\"")
+                    .append(longFormat)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("DBFormat=\"")
+                    .append(dbFormat)
+                    .append("\" )");
+            } // http://jira.dotmarketing.net/browse/DOTCMS-2869
+              // else if (field.getFieldType().equals(Field.FieldType.CUSTOM_FIELD.toString())){
+              // sb.append("#set( $" + field.variable() + "Code=\"" +
+              // UtilMethods.espaceForVelocity(field.getValues()) + "\" )");
+              // }//http://jira.dotmarketing.net/browse/DOTCMS-3232
+            else if (field instanceof HostFolderField) {
+                if (InodeUtils.isSet(content.getFolder())) {
+                    sb.append("#set( $ConHostFolder='")
+                        .append(content.getFolder())
+                        .append("' )");
+                } else {
+                    sb.append("#set( $ConHostFolder='")
+                        .append(content.getHost())
+                        .append("' )");
+                }
+            }
+
+            else if (field instanceof CategoryField) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "else" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)